.travis.yml

```diff
@@ -25,8 +38,39 @@ env:
     - TARGET_PLATFORM=ubuntu/bionic
     - TARGET_PLATFORM=ubuntu/xenial
 before_install:
-  - export PACKAGE_ENCRYPTION_KEY=verysecret
-  - git clone -b v0.7.14 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b simplify-branching --depth 1 https://github.com/citusdata/tools.git
```

Comment:
This should be changed once https://github.com/citusdata/tools/pull/62 is merged

Reply:
we should not forget to revert this back before merging.